bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/NetworkRequestInstrumentationCallback.kt

```diff
@@ -0,0 +1,5 @@
+package com.bugsnag.android.performance
+
+fun interface NetworkRequestInstrumentationCallback {
+    fun onNetworkRequest(info: NetworkRequestInfo)
+}
```

Comment:
It would be good to get a little code documentation on this, maybe with a short example snippet to explain how to use it?